lib/tasks/osc.rake

```diff
@@ -74,6 +74,21 @@ namespace :osc do
     end
   end
 
+  def check_IDs_in_changes_file!
+    # Checking if any new bugzilla,fate,... IDs are defined
+    # in the *.changes file
+    puts "Checking IDs in *.changes file" if verbose
+    Dir.chdir(osc_checkout_dir) do
+      cmd = "osc -A '#{obs_api}' diff *.changes | grep \"bnc#[0-9]\\|fate#[0-9]\\|boo#[0-9]\\|bsc#[0-9]\\|bgo#[0-9]\""
+      puts cmd if verbose
+      `bash -c '#{cmd}'`
+      return if $?.success?
+    end
+    puts "Stopping, missing new bugzilla or fate entry in the *.changes file"
+    puts "Valid entries are bnc#<number>, fate#<number>, boo#<number>, bsc#<number>, bgo#<number>"
```

Comment:
ok, fine. Regarding CVE I think we should stick to original plugin approach and ideally also reference that plugin, so we know where to check if it is changed as it is authoritative source which can decline request.